ci/travis-37-numpydev.yaml

```diff
@@ -13,5 +13,5 @@ dependencies:
     - "git+git://github.com/dateutil/dateutil.git"
     - "-f https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com"
     - "--pre"
-    - "numpy"
+    - "numpy<=1.16.0.dev0+20181015190246"
```

Comment:
these are just strings, so the < part won't work here, just pin = for now

Reply:
That grabbed NmPy commit 54985e3, so it seemed to have worked.

Reply:
https://travis-ci.org/pandas-dev/pandas/jobs/442114100#L727